app/src/main/AndroidManifest.xml

```diff
@@ -11,8 +11,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:usesCleartextTraffic="${usesCleartextTraffic}"
```

Comment:
Hey,
Why is the change in line 14 & 15 needed? Is it just changing the order in which attributes are listed? If so, can we do without this change?

Reply:
It was just autoformatting according to Java and XML coding style standards. I deleted it in this file.